test/project_types/script/layers/application/build_script_test.rb

```diff
@@ -45,6 +46,8 @@
       script_project
         .stubs(:language)
         .returns(library_language)
+
+      Script::Layers::Infrastructure::MetadataRepository.any_instance.stubs(:get_metadata).returns(metadata)
```

Comment:
Do you think we should return a FakeMetadataRepository here instead? We have a couple other fake repos in the CLI

Reply:
I'm not sure at what point it becomes useful to create a Fake. In this case, the functionality of the MetadataRepository is quite limited (1 method) so I would say it's not worth doing yet. WDYT?

Reply:
Both you and Andrew had the same comment so I went ahead and implemented it 😂